types/types.d.ts

```diff
@@ -92,10 +92,7 @@ declare namespace FastifySessionPlugin {
 export class MemoryStore implements FastifySessionPlugin.SessionStore {
   constructor(map?: Map<string, any>);
   set(sessionId: string, session: any, callback: (err?: Error) => void): void;
```

Comment:
why is `session` typed as `any` instead of `Session`?

Reply:
This is inherited from previous code. We can fix it in another PR.